test/cmdlineTests/evm_to_wasm_break/output

```diff
@@ -68,15 +68,15 @@ object "object" {
             let r1_1, carry_2 := add_carry(x1, y1, carry_1)
             r1 := r1_1
         }
-        function iszero_200_872_1499(x1, x2, x3, x4) -> r1, r2, r3, r4
+        function iszero(x1, x2, x3, x4) -> r1, r2, r3, r4
         {
             r4 := i64.extend_i32_u(i64.eqz(i64.or(i64.or(x1, x2), i64.or(x3, x4))))
         }
-        function eq_201_873_1500(x1, x2, x3, x4, y1, y2, y3, y4) -> r1, r2, r3, r4
+        function eq(x1, x2, x3, x4, y1, y2, y3, y4) -> r1, r2, r3, r4
```

Comment:
Oh wow, this is still a bug.

Reply:
Ah no, it is not, because the builtin does not exist in ewasm.

Reply:
Looking at `libyul/backends/wasm/polyfill/Comparison.yul` a polyfill `function eq(x1, x2, x3, x4, y1, y2, y3, y4) -> r1, r2, r3, r4 ` is defined. I guess it works because `NameDispenser` is used within `EVMToEwasmTranslator::run(..)`.

Reply:
Ah, you meant the builtin defined by the dialect?